src/support/rand.c

```diff
@@ -57,6 +59,23 @@ __wt_random_init(WT_RAND_STATE volatile *rnd_state) WT_GCC_FUNC_ATTRIBUTE((visib
     *rnd_state = rnd;
 }
 
+/*
+ * __wt_random_init_custom_seed --
+ *     Initialize the state of a 32-bit pseudo-random number with custom seed.
+ */
+void
+__wt_random_init_custom_seed(WT_RAND_STATE volatile *rnd_state, uint64_t v, uint32_t w, uint32_t z)
```

Comment:
I think this should be:
```
void
__wt_random_init_custom_seed(WT_RAND_STATE volatile *rnd_state, uint64_t v)
```
The reason is that v is a union with (w,z).  `w` and `z` are each 32 bits and when put together in memory can be read as the 64 bit quantity `v`.  That's done for coding convenience.  So setting them all is redundant.
```

Reply:
Fixed